src/content/ui-api/kubectl-gs/get-appcatalogs.md

```diff
@@ -89,13 +89,13 @@ Here we document the flags that have a particular meaning for the `get appcatalo
 
 To inspect a cluster's main custom resource in YAML notation, add the `--output yaml` flag (or `-o yaml` in short) to the command.
 
-The following example command would print the main resource for cluster `ab12c`. It would return the [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}}) resource.
+The following example command would print the main resource for cluster `ab12c`. It would return the [App Catalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}}) resource.
```

Comment:
Same here

```suggestion
The following example command would print the main resource for cluster `ab12c`. It would return the [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}}) resource.
```